scripts/python/lib/config.py

```diff
@@ -2332,10 +2333,12 @@ def lookup_interface_iface(self, if_label):
             if interface.label == if_label:
                 if self.CfgKey.IFACE in interface:
                     return interface[self.CfgKey.IFACE]
+                elif self.CfgKey.DEVICE in interface:
+                    return interface[self.CfgKey.DEVICE]
                 else:
                     raise UserException(
-                        'No \'iface\' key defined in interface with label=%s' %
-                        interface.label)
+                        'No \'iface\' or \'DEVICE\' key defined in interface '
+                        'with label=%s' % interface.label)
```

Comment:
@rayjh I did end up making a small change to config.py. It's so minor I didn't end up submitting it as it's own commit.